composer.json

```
@@ -38,7 +38,7 @@
     },
     "require-dev": {
         "az-digital/az-quickstart-dev": "~1",
-        "drupal/core-dev": "^10.2"
+        "drupal/core-dev": "^10.2.0"
```

I think it needs to be this to allow the patch version to go up.
```suggestion
        "drupal/core-dev": "~10.2.0"
```

---

Don't think so.